src/Rules/Doctrine/ORM/EntityColumnRule.php

```diff
@@ -64,9 +68,6 @@ public function processNode(Node $node, Scope $scope): array
 		}
 
 		$className = $class->getName();
-		if ($objectManager->getMetadataFactory()->isTransient($className)) {
```

Comment:
I'm trying to figure out internally why the embeddables are considered transient, but removal of this check causes no test failures

Reply:
Also classes not managed by Doctrine will have an exception thrown a bit later while trying to get their metadata and will return early from the rule with no errors, so I believe the intent is preserved

Reply:
I've created an issue in ORM but I think we can continue without waiting for its resolution https://github.com/doctrine/orm/pull/8006